Jaewon

jwkim395 opened the pull request:
#1  링크드 리스트 템플릿화

kimjw395 committed:
-- 연산자(전위, 후위) 구현
end() 구현
begin() 수정
복사, 이동 연산자 구현
복사, 이동 생성자 구현
main 함수에 swap(이동 연산자 사용하여) 구현
CopyShallow함수와 CopyDeep 함수 구현
  - 이동, 복사 생성자 구현
  - 대입, 이동 대입 연산자 구현
  - CopyShallow함수와 CopyDeep 함수 구현

kimjw395 committed:
CList 탬플릿화

kimjw395 committed:
주석 추가 작성

jeontigger commented on Jaewon/Project1/Project1/TemplateCList.h:
```diff
@@ -0,0 +1,146 @@
+			head->prev = newNode;
+		head = newNode;
+		++dCnt;
+	}
```
복붙을 해오는 과정중에 new 파라미터가 잘못된거같은데 제대로 동작이 되나?

jwkim395 replied:
복사 생성자, 이동 생성자, 대입연산자, 이동대입연산자 미구현 항목들 구현하였으며
 pushBack, pushFront의 new 파라미터 수정 완료하였습니다.

jeontigger reviewed (commented):
복사 생성자와 이동 생성자, 대입연산자와 이동대입연산자가 구현이 안되어있음!

kimjw395 committed:
1. 데이터 삽입 -완
2. 전위, 중위, 후위 탐색 출력, 깊이 출력
 - 전,중,후 완 / 깊이는 아직
3. 데이터 검색 - 완
4. 데이터 삭제 - 아직

kimjw395 committed:
링크드 리스트 템플릿화 수정 사항
 - 미구현 항목 구현완료
     - 복사 생성자
     - 이동 생성자
     - 대입 연산자
     - 이동대입 연산자
- pushBack, pushFront의 new 파라미터 수정
